MyOwnORM/MyOwnORM/Reflection/CustomDbSetReflection.cs

```diff
@@ -0,0 +1,514 @@
+﻿using Microsoft.CSharp.RuntimeBinder;
+using MyOwnORM.Attributes;
+using MyOwnORM.Reflection.Helpers;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Reflection;
+using System.Reflection.PortableExecutable;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyOwnORM.Reflection
```

Comment:
Reflection is not good name for folder. This class is service or helper

Reply:
Fixed